tvb/simulator/plot/timeseries_interactive.py

```diff
@@ -61,6 +61,8 @@
 import numpy
 import pylab
 import matplotlib.widgets as widgets
+from matplotlib import rcParams
+from six import string_types
```

Comment:
We should be py3 compatible only. 
Maybe we can drop this import?

Reply:
Indeed, this can be dropped.